src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -292,3 +292,19 @@ fn fix_code_blocks(html: String) -> String {
         format!("<code{before}class=\"{classes}\"{after}>", before=before, classes=classes, after=after)
     }).into_owned()
 }
+
+fn add_playpen_pre(html: String) -> String {
+    let regex = Regex::new(r##"((?s)<code[^>]?class="([^"]+)".*?>.*?</code>)"##).unwrap();
+    regex.replace_all(&html, |caps: &Captures| {
+        let text = &caps[1];
+        let classes = &caps[2];
+
+        if classes.contains("language-rust") && !classes.contains("ignore") {
```

Comment:
what about should_panic? i can't decide whether not having a run button or having a run button that then causes a panic would be more confusing, just want to make sure we consider it :)